waspc/src/Wasp/AppSpec/App/EmailSender.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+
+module Wasp.AppSpec.App.EmailSender
+  ( EmailSender (..),
+    EmailProvider (..),
+    EmailFrom (..),
+  )
+where
+
+import Data.Data (Data)
+
+data EmailSender = EmailSender
+  { provider :: EmailProvider,
+    defaultFrom :: EmailFrom
+  }
+  deriving (Show, Eq, Data)
+
+data EmailProvider = SMTP | SendGrid | Mailgun
+  deriving (Eq, Data)
+
+instance Show EmailProvider where
+  show SMTP = "smtp"
+  show SendGrid = "sendgrid"
+  show Mailgun = "mailgun"
+
+data EmailFrom = EmailFrom
```

Comment:
I think I saw you calling this `EmailFromField` in other pieces of code? What is better, `EmailFrom` or `EmailFromField`? I think I like the second one a bit better, it seems to provide a bit more information.

Reply:
Agreed 👍